docs/source/integrations/ios-fairplay-sdk.md

```diff
@@ -1,138 +1,140 @@
-    		private var token = ""
-   		private var assetName = ""
- 
-● To initialise via view controller on device, populate the URL, Token, and Asset Name fields with valid values. For ease of use, pasting into the URL field in the form URL + space + Token will auto populate both fields correctly. At runtime, any values in the URL and Token fields will override any hard coded values. 
-
-### Limitations ###
+let player = AVPlayer(playerItem: playerItem)
+
+3. Finally, present your player:
+   self.player = player let playerController = AVPlayerViewController() playerController.delegate = self as? AVPlayerViewControllerDelegate self.present(playerController, animated: true, completion: nil) playerController.view.frame = self.view.frame playerController.player = self.player player.play()
+   DEMO
+   With the VUDRMFairPlay framework installed, the example demo application can be initialised either via source code in the IDE, or using field entry on device.
+   ● To initialise via code in IDE, before running, simply replace the empty ViewController.swift values with your own valid values.
+   private var assetURL = URL(string: "")
+   private var token = ""
+   private var assetName = ""
+
+● To initialise via view controller on device, populate the URL, Token, and Asset Name fields with valid values. For ease of use, pasting into the URL field in the form URL + space + Token will auto populate both fields correctly. At runtime, any values in the URL and Token fields will override any hard coded values.
+
+### Limitations
 
 ● The current example application demonstrates a single instance of offline playback using an AVAssetDownloadURLSession with a URLSessionConfiguration to create an AVAssetDownloadTask for that asset. The example is not able to handle multiple assets. Users are expected to develop a method of mapping instances to assets within their application which will avoid multiple downloads of the same asset and will enable management of assets.
-The example includes legacy methods for online playback only. The use of these methods in conjunction with the default method may cause unexpected results and is not recommended. 
-● The SDK does not support playback of protected content when running in the iOS Simulator. Attempting to do so will result in an invalid KEYFORMAT error and the content will not load. 
+The example includes legacy methods for online playback only. The use of these methods in conjunction with the default method may cause unexpected results and is not recommended.
+● The SDK does not support playback of protected content when running in the iOS Simulator. Attempting to do so will result in an invalid KEYFORMAT error and the content will not load.
 
-### Known Issues ###
+### Known Issues
 
-● No known issues at this time. If you believe you have found an issue, please contact us at support@vualto.com 
+● No known issues at this time. If you believe you have found an issue, please contact us at support@vualto.com
```

Comment:
Change to `<support@vualto.com>` to enable as an email link

Reply:
any changes to this document will have to be done by Adam